driver/annotations.md

```diff
@@ -2,7 +2,7 @@
 
 # Intro
 
-Once you have written the [AST parser](sdk.md) the next step is to write
+Once you have written the [native code to AST parser](sdk.md) the next step is to write
```

Comment:
As you prefer, I just find *code to AST* redudant, while using _native parser_ is consistent.